guess-ai/src/engine.rs

```diff
@@ -388,7 +416,7 @@ impl GuessAiEngine {
 
             self.hints.push(hint.hint.clone());
 
-            todo!("Add a client for social media to post the hint");
+            // todo!("Add a client for social media to post the hint");
```

Comment:
why commenting this ?

Reply:
Same as above, didn't want the code to crash.
Uncommented.